packages/fxa-react/components/Portal/index.tsx

```diff
@@ -23,6 +55,12 @@ const Portal = ({ id, children }: PortalProps): React.ReactPortal | null => {
       let el = document.getElementById(id);
       if (el && el.children.length === 1) {
         el.remove();
+
+        if (id === 'modal') {
+          resetA11yOnAdjacentElementsAndBody(
+            document.querySelectorAll('body > div:not(#modal)')
```

Comment:
Given the specificity and re-use, `body > div:not(#modal)` might be better suited to a constant